app/scripts/services/Citations.svc.js

```diff
@@ -1,14 +1,52 @@
 'use strict';
 
-angular.module('yourStlCourts').factory('Citations', function ($resource) {
+angular.module('yourStlCourts').factory('Citations', function ($resource,$q) {
   var CitationResource = $resource('citations/:id');
 
   function find(parameters) {
-    return CitationResource.get(parameters).$promise;
+    var deferred = $q.defer();
+    CitationResource.get(parameters,function(citations){
```

Comment:
So when I do this I get the following browser error: TypeError: Citations.find(...).then is not a function
    at Object.ctrl.findTicket (ticketFinder.ctrl.js:65)
If I read (https://docs.angularjs.org/api/ngResource/service/$resource) correctly, it states: "It is important to realize that invoking a $resource object method immediately returns an empty reference (object or array depending on isArray). Once the data is returned from the server the existing reference is populated with the actual data."

So it is no longer returning a promise but the actual data itself.  
Isn't it cleaner to keep our existing code intact by continuing to return a promise by using $q.defer?  It also allows ticketFinder.ctrl to look at the result and act on it.